geoutils/geoviewer.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+#coding=utf-8
+"""
+geoutils.geoviewer provides a toolset for plotting raster and vector data
```

Comment:
command line executables should probably live somewhere else than packages for clarity (/bin?)

Reply:
Good point... It all depends what we put in this geoviewer.py. We could have a library where we put a bunch of useful functions for displaying and plots options, that could be accessed as a submodule. Then we could have a simple CLI tool with some of the options. But how to call them both in that case??